src/components/pages/Photos/ImageBlock.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import CameraNavbar from './cameraNavbar';
+import './PhotoPage.css';
+
+export default class ImageBloack extends Component {
+  render() {
+    return (
+      <div className="rover-image">
+        <img className="camera-capture" src="../../../assets/images/logo.svg" alt="Mars" />

```

Comment:
The way to reference a photo src in React is to `import` the file as a variable and then use that variable as the src property.

See `TempPage.js` lines 2 and 9 for an example!